src/entities/Profile.ts

```diff
@@ -36,8 +47,12 @@ export default class Profile {
 	})
 	public course!: string;
 
-	@Column({ type: 'integer' })
-	public yearOfStudy!: number;
+	@IsString()
```

Comment:
I've just had a look and class-validator also has an `@IsEnum` decorator! I think this is fine for now, but I'll make a separate issue for updating yearOfStudy and Course to use this decorator, and to also update APIProfile to have course as an enum instead of just a string.